Libplanet/Net/NetMQTransport.cs

```diff
@@ -178,13 +202,16 @@ internal class NetMQTransport : ITransport
                 _endPoint,
                 _publicIPAddress);
 
+        /// <inheritdoc />
         public IEnumerable<BoundPeer> Peers => Protocol.Peers;
 
+        /// <inheritdoc cref="ITransport.LastMessageTimestamp"/>
         public DateTimeOffset? LastMessageTimestamp { get; private set; }
 
         /// <summary>
         /// Whether this <see cref="NetMQTransport"/> instance is running.
         /// </summary>
+        /// <returns>Boolean value indicates whether the instance is running.</returns>
```

Comment:
In my opinion it is enough to comment only *whether the instance is running.*
What I want to say really, is that I wonder should we comment it is bool-typed.

Reply:
This does not strongly indicates that the return value is a bool-typed value. Other documents we wrote has similar sentence. For instance:
```
/// <returns>An awaitable task without value.</returns>
```
from Swarm<T>.StartAsync() is not to indicate that it is Task-typed value.